### DISCUSSION TITLE
How to configure its terminal to work with chafa?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#1334] [Q&A]
[chafa](https://hpjansson.org/chafa/) is an awesome tool to show images on terminals, but I found that images are blurred greatly if I use the built-in terminal of LazyVim invoked by `<leader>ft` or `<leader>fT`.

<img width="787" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/38e529f5-4e09-4373-88f4-1c776f668fe3">

According to docs of `chafa`, it seems that it fails to detect the terminal's size:

> Set maximum output dimensions in columns and rows. By default this will be the size of your terminal, or 80x25 if size detection fails. 

But even if I specify the size, the image is still blurred:

<img width="377" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/9a0219fa-6c96-45d3-b24f-6def1604eb36">

So, how to configure LazyVim's terminal to work with chafa?

----

I am using nvim 0.9.1 on macOS with [kitty](https://sw.kovidgoyal.net/kitty/). The following is the one that I show the image directly (not in nvim):

<img width="311" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/c0bdc261-85df-4e32-8633-0d3da2dccb9f">

**Comment by xu8533** [comment]
same issue, any resolve?
